### PR TITLE
Two Sigma lenses added for Nikon

### PR DESCRIPTION
**Pull request by norbertwg**
Following lenses added for Nikon (refers to #2779):
ID 191: Sigma 60–600mm F4.5-6.3 DG OS HSM | Sports
ID 193: Sigma 14-24mm f/2.8 DG HSM Art Lens

### CODE REVIEW

File: src/nikonmn_int.cpp [hunk 1]
```diff
@@ -2809,6 +2813,9 @@ std::ostream& Nikon3MakerNote::printLensId(std::ostream& os, const Value& value,
       {0xA7, 0x49, 0x80, 0xA0, 0x24, 0x24, 0x4B, 0x06, 0x03, 0x00, 0x00, "Sigma", "", "APO 200-500mm F2.8 EX DG"},
       {0x48, 0x3C, 0x8E, 0xB0, 0x3C, 0x3C, 0x4B, 0x02, 0x03, 0x00, 0x00, "Sigma", "595555",
        "APO 300-800mm F5.6 EX DG HSM"},
+      {0xBF, 0x38, 0x56, 0xA6, 0x34, 0x40, 0x4B, 0x4E, 0x00, 0x00, 0x00, "Sigma", "",
+       "60-600mm F4.5-6.3 DG OS HSM | Sports"},
```
Comment (kmilos) [inline src/nikonmn_int.cpp:2817]:
```suggestion
       "60-600mm F4.5-6.3 DG OS HSM | S"},
```

File: src/nikonmn_int.cpp [hunk 2]
```diff
@@ -2809,6 +2813,9 @@ std::ostream& Nikon3MakerNote::printLensId(std::ostream& os, const Value& value,
...
+      {0xC1, 0x48, 0x24, 0x37, 0x24, 0x24, 0x4B, 0x46, 0x00, 0x00, 0x00, "Sigma", "", "14-24mm f/2.8 DG HSM Art Lens"},
```
Comment (kmilos) [inline src/nikonmn_int.cpp:2818]:
```suggestion
      {0xC1, 0x48, 0x24, 0x37, 0x24, 0x24, 0x4B, 0x46, 0x00, 0x00, 0x00, "Sigma", "", "14-24mm F2.8 DG HSM | A"},
```

**Review by kmilos — CHANGES_REQUESTED** [review]
Let's try to keep the lens string style and naming scheme consistent within particular makernote file for a particular brand.

Matching exiftool is also desirable: https://exiftool.org/TagNames/Nikon.html

**Comment by norbertwg** [comment]
@kmilos thanks for this hint

**Review by kmilos — APPROVED** [review]

**Pull request merged**